tests/__init__.py

```diff
@@ -27,9 +27,9 @@
 from ratbag_emu.protocol.base import MouseData
```

Comment:
This is a client for ratbag-emu. It is not required, however it will save use from dealing with the HTTP request directly. The client will assert the error code and return the actual data that we want. IMO it makes protocol tests much more readable. The reason it has unused functions it because it was synced from the libratbag test suite.